cookbooks/aws-parallelcluster-platform/spec/unit/recipes/openssh_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+describe 'aws-parallelcluster-platform::openssh' do
+  for_all_oses do |platform, version|
+    context "on #{platform}#{version}" do
+      cached(:vpc_cidr_list) { '(cidr1 cidr2)' }
+      cached(:chef_run) do
+        runner(platform: platform, version: version).converge(described_recipe)
+      end
+      cached(:node) { chef_run.node }
+
+      before do
+        mock_get_vpc_cidr_list(vpc_cidr_list)
+      end
+
+      it 'installs the ssh target checker with the correct attributes' do
```

Comment:
I would also check the file contents (see the documentation [here](https://github.com/chefspec/chefspec#render_file)). You can find some examples in ParallelCluster code as well (for instance, you might try to take a look at `isolated_install_spec.rb`).

Reply:
Sure. Will add this. Thanks for the file.